### DISCUSSION TITLE
GitHub Markdown Editor

### DISCUSSION THREAD

**Discussion by Stecyk** [community/community#23786] [New to GitHub]
<p>Hello,</p>
<p>Do you know who I can contact regarding the Markdown Editor found on this page: <a href="https://jbt.github.io/markdown-editor/" rel="nofollow noopener">https://jbt.github.io/markdown-editor/</a> ?</p>
<p>The editor does not correctly display the results for example 51: (Because I am limited to two links, please see the link in the code section below.)</p>
<p>In the example, the resulting display should be an H1. Instead, it is plain text.</p>
<pre><code class="lang-auto">Here's my editor session: 
https://jbt.github.io/markdown-editor/#jZFNasNADIX3OsWDLAqhsZNCNoYs0ybQ7nqBcazYgycjd37quKfv2GmaRShUMAyIp6dP0gwvOuxiiTfl2kp6i+2Z3UF7Jtqe1akzjPWqQBNC54s8r3VoYpn9fAc55fXxlM/4Il2sV6iEPawE9OJadE46dmZAyQcVPSM06fE5wDcSTZXyUBa7VUb0LIJ5qRyV6mtOmxREtP9VGpEWRrejh/ap0EgPbSfHSvvOqKEgmuFqg8mG6JXDg0dwA6LXtoZCpXyTKn1gVUGOkwN/RGXgdW3vSBYpks97UqlSPhmOfTQBCaJ3khyDSIb9lBgHH+d5yv5CZxzFJPSEMuLe89LeBk4tQlKY4RF8PcNyWqkv8O9rLG+jXBa6uWz1llxcg74B
</code></pre>
<p>Here’s the code from the Markdown Editor:</p>
<pre><code class="lang-auto"># GitHub Markdown Exercise

Example 51: https://github.github.com/gfm/#example-51 does not work properly because the text should be an H1.

Foo *bar
baz*
====

It should look like this below in the display:

# Foo *bar baz*

Let's try using a dash instead of the equal sign.

Foo *bar
baz*
----

The above result is wrong too. It is not an H2.

It should look like the following:

## Foo *bar baz*

Interestingly, example 50 works:  https://github.github.com/gfm/#example-50

Foo *bar*
=========

Foo *bar*
---------
</code></pre>
<p>I hope this information is helpful.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/7/792a42fc4b4f49d61c7a51de29548d35cc05462e.jpeg" data-download-href="/uploads/short-url/hhSjMGL2rFrQtjJSSA5iE1jeo4S.jpeg?dl=1" title="2020-06-08_9-48-35-GitHubEditor" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/31547847/181079336-22e7aab1-8634-402d-9331-b47e01fc59ad.jpeg" alt="2020-06-08_9-48-35-GitHubEditor" data-base62-sha1="hhSjMGL2rFrQtjJSSA5iE1jeo4S" width="690" height="379" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/7/792a42fc4b4f49d61c7a51de29548d35cc05462e_2_690x379.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/7/792a42fc4b4f49d61c7a51de29548d35cc05462e_2_1035x568.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/7/792a42fc4b4f49d61c7a51de29548d35cc05462e_2_1380x758.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/7/792a42fc4b4f49d61c7a51de29548d35cc05462e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2020-06-08_9-48-35-GitHubEditor</span><span class="informations">1920×1057 204 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Sincerely,<br>
Kevin</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/stecyk">@Stecyk</a>, Welcome to the community! Hi,</p>
<p>It sounds like you’re looking for help with a specific <a href="https://github.com/jbt/markdown-editor">project hosted on GitHub</a>. I’d recommend checking the project’s <a href="https://github.com/jbt/markdown-editor/blob/58aa8bfe18c5a30af2c6fb08221b00b841688dd9/README.md">README</a> for instructions on how to best get support; a lot of projects will have you submit issues to their repo or provide email addresses for contact.</p>
<p>Best of luck!</p>

**Comment by Stecyk** [comment]
<p><a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a></p>
<p>Thank you. You pointed me to the correct page. I did not realize it at first.</p>
<p>Sincerely,<br>
Kevin</p>
